logistics_project/apps/malawi/templatetags/malawi_warehouse_tags.py

```diff
@@ -3,16 +3,17 @@
 from django.core.exceptions import ObjectDoesNotExist
 from logistics.models import SupplyPoint
 from logistics.templatetags.logistics_report_tags import r_2_s_helper
+from logistics.util import config
```

Comment:
this should work fine, though I've stopped importing from here and switched to importing the malawi config directly just to make my IDE be able to resolve the references easier